perf/stability/alertmanager/templates/alertmanager-service.yaml

```diff
@@ -0,0 +1,38 @@
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: alertmanager-main
```

Comment:
Not a big concern, but does the operator not handle this?

Reply:
You mean creating the rbac stuff? Seems like it does not, but I can double check

Reply:
is this for running the actual alert manager part of Prometheus or is that just part of core Prometheus and this is for our webhook golang program?

Reply:
it is for alert manager part of Prometheus